apps/question-the-docs/backend/ai/utils/github.py

```diff
@@ -53,41 +73,20 @@ def download_and_decode(url):
     return base64.b64decode(blob['content'])
 
 
-def save_github_md_files_locally(repo_details):
-    print(f"Downloading files from GitHub for {json.dumps(repo_details)}")
-    owner = repo_details['owner']
-    name = repo_details['repo']
-    branch = repo_details['branch']
-    documentation_location = repo_details['documentation_location']
+def save_github_md_files_locally(repo):
+    owner, name, branch, documentation_location = REPOS[repo].values()
```

Comment:
Now, this is really playing with fire here. 😛  You are replying on the "order" of the entries in a dict, without telling anyone.

Someone could secretly break this code by sorting the entries in the dictionaries!

All the more reason for a dataclass.